src/core/packages/test-helpers/so-type-serializer/src/extract_migration_info.ts

```diff
@@ -29,11 +36,13 @@ export interface SavedObjectTypeMigrationInfo {
 
 export interface ModelVersionSummary {
   version: string;
+  modelVersionHash: string;
```

Comment:
The hash shouldn't change once a modelVersion has been declared. Comparing the hash could be used instead of comparing full model versions, although it only indicates _something_ changed. To see what changed, we still need to compare model version properties individually.